src/sentry/api/endpoints/project_rules.py

```diff
@@ -91,15 +91,19 @@ def post(self, request: Request, project) -> Response:
                     slow_rules += 1
                     break
 
-        if new_rule_is_slow and slow_rules >= settings.MAX_SLOW_CONDITION_ISSUE_ALERTS:
-            return Response(
-                {
-                    "conditions": [
-                        f"You may not exceed {settings.MAX_SLOW_CONDITION_ISSUE_ALERTS} rules with this type of condition per project.",
-                    ]
-                },
-                status=status.HTTP_400_BAD_REQUEST,
-            )
+        if new_rule_is_slow:
+            max_slow_alerts = settings.MAX_SLOW_CONDITION_ISSUE_ALERTS
+            if features.has("organizations:more-slow-alerts", project.organization):
+                max_slow_alerts = settings.MAX_SLOW_CONDITION_ISSUE_ALERTS * 2
```

Comment:
Not sure if we wanted to do 2x, +100, or hardcode 200?

Reply:
Let's just do +100, that way it doesn't go crazy if we increase the base limit at some point. Might want to make that a setting/constant too